tac/gui/dashboards/agent.py

```diff
@@ -83,7 +83,6 @@ def __init__(self, agent_name: str,
 
     def init(self):
         """Re-initiate the AgntDashboard."""
```

Comment:
can you fix the typo in the docstring? > AgentDashboard

Reply:
yes